packages/statemanager/src/stateManager.ts

```diff
@@ -902,7 +902,7 @@ export class DefaultStateManager implements EVMStateManagerInterface {
         32
       )
       if (!equalsBytes(reportedValue, storageValue)) {
-        throw new Error('Reported trie value does not match storage')
+        throw new Error(`Reported trie value does not match storage, key: ${stProof.key}`)
```

Comment:
Should we also report the hex values of the reportedValue and storageValue here?

Reply:
Yup, will add